documentation/book/con-maintenance-time-window-definition.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// assembly-maintenance-time-windows.adoc
+
+[id='con-maintenance-time-window-definition-{context}']
+= Maintenance time window definition
+
+You configure maintenance time windows by entering an array of strings in the `Kafka.spec.maintenanceTimeWindows` property.
+Each string is a {CronExpression} interpreted as being in UTC (Universal coordinated time, which for practical purposes is the same as Greenwich Mean Time).
+
+The following example would configure a single maintenance time window which starts at midnight and ends at 01:59am (UTC), on Sundays, Mondays, Tuesdays, Wednesdays and Thursdays:
+
+[source,yaml,subs="attributes+"]
+----
+# ...
+maintenanceTimeWindows:
+  - "* * 0-1 ? * SUN,MON,TUE,WED,THU *"
+# ...
+----
+
+In practice, maintenance windows should be set in conjunction with the `Kafka.spec.clusterCa.renewalDays` and `Kafka.spec.clientsCa.renewalDays` properties of the `Kafka` resource, to ensure that the necessary CA certificate renewal can be completed in the configured maintenance time windows.
+
+NOTE: {ProductName} does not schedule maintenance operations exactly according to the given windows. Instead it checks on each reconciliation whether a maintenance window is currently "open".
```

Comment:
Instead it checks on each reconciliation > Instead, for each reconciliation, it checks whether...